docs/transformations.rst

```diff
@@ -23,6 +23,18 @@ Code example to make images inline:
     >>> message.html
     u'<html><body><img src="cid:promo.png"></body></html>'
 
+To inline CSS using `Premailer <https://github.com/peterbe/premailer>`_:
+
+.. code-block:: python
+
+    >>> html = '''<style>h1 { color: red }</style>
+    ...           <h1>Hello world!</h1>'''
+    >>> message = emails.Message(html=html)
+    >>> message.transformer.premailer.transform()
```

Comment:
`Premailer` is a great library, but using it's abstractions here looks a little confusing for me.  

Supposed way to do basic transformations is `Transformer.load_and_transform`:

```
message = emails.Message(html='''<style>h1 { color: red }</style>
         <h1>Hello world!</h1>''')
message.transformer.load_and_transform()
message.transformer.save()
print(message.html)
```